modules/reference/pages/feature/jwtSso/examples.adoc

```diff
@@ -31,8 +31,9 @@ You can use a mechanism other than JWT cookies for authentication, as the JWT co
 
 ===  Configure an authentication filter
 
-You can configure an authentication filter to secure requests for accessing protected resources that are used with JWT SSO for authentication.
-The `jwtSso` feature supports authentication filters.
+You can configure an authentication filter to secure requests for accessing protected resources that are configured with JWT SSO for authentication.
+If the request meets the criteria that are configured in the JWT SSO authentication filter, then you can authenticate with the JWT to access the protected resource.
+Conversely, if the request does not meet the criteria that are configured in the JWT SSO authentication filter, then you are prompted to provide login credentials.
```

Comment:
Conversely, if the request does not meet the criteria that are configured in the JWT SSO authentication filter, then the user is prompted to provide login credentials.